type-extensions.ts

```diff
@@ -13,6 +13,10 @@ declare module "hardhat/types/runtime" {
       | "arbitrumone"
       | "arbitrumsepolia"
       | "zksyncsepolia"
-      | "zksyncmainnet";
+      | "zksyncmainnet"
+      | "basesepolia"
+      | "basemainnet"
+      | "opsepolia"
+      | "opmainnet";
   }
 }
```

Comment:
Could you please add new network [here](https://github.com/VenusProtocol/vips/blob/main/src/types.ts#L3)